ncm-aiiserver/src/test/resources/aii-options.pan

```diff
@@ -0,0 +1,30 @@
+@{
+Profile for testing main aii-server optioins
+@}
+object template aii-options;
+
+include 'components/aiiserver/schema';
```

Comment:
just include the config like http://www.quattor.org/documentation/2017/02/10/components-spring17-cleaning.html#add-a-unittest